cmd/config/internal/commands/cmdcreatesetter.go

```diff
@@ -48,6 +48,9 @@ func NewCreateSetterRunner(parent string) *CreateSetterRunner {
 	set.Flags().MarkHidden("partial")
 	set.Flags().StringVar(&setterVersion, "version", "",
 		"use this version of the setter format")
+	set.Flags().StringVar(&r.CreateSetter.SchemaPath, "schema-path", "",
```

Comment:
I'm not sure about the name of this flag. Schema is a bit vague, so maybe we could find a more specific name. My PR in the kpt repo to allow fetching the openAPI schema from the cluster/file actually uses the same name for a flag (https://github.com/GoogleContainerTools/kpt/pull/612/files#diff-58463f45efceb467df8560e5fda87a82R133). I'm open to input on other names on that flag too.

Reply:
As the tool heavily uses openAPI schema all across, it should be evident and intuitive to users that Schema refers to OpenAPI schema. The flag name is short. And I came with this name here without knowing that it is in your PR, which means it is intuitive. So I would go with this name.

Reply:
We are using openAPI in many places, so we just need to make sure it is clear to users how the different schemas are being used. 
So the flag in my PR refers to the entire openAPI schema kpt would be using and it is a persistent flag that will be available on all subcommands. Thus, we need to have different names for letting users specify the path to the Kubernetes openAPI schema and the schema for adding validation to setters.
The flag in the other PR refers specifically to how we fetch the k8s schema, so it might be better to use a name like `k8s-schema-path` for that flag.
For this one, we are not really asking users to add a root schema here right? It is just a subset of the schema, in particular schema properties for defining types. I think we should try to see if we can come up with a name that reflects this. Also, should we any validation when we create setters that the content of the provided schema here makes sense? I think right now it only needs to be valid json.

Reply:
@mortent From our offline conversation, we can go ahead with name schema-path. 
Regarding validations, I would leave it open and get more feedback from users first and go from there. If user inputs some invalid field ex: {"foo": "bar"}, the json will still be accepted but set would not parse it into openAPI schema object, so the this means there are no validations to be performed. Let me know your thoughts.

Reply:
I have updated the flags in the kpt PR to use `k8s-schema-path`, which I think is a better name for that use case than `schema-path`. So I think we can keep the `schema-path` name here. Agree that we can look at validating the provided schema later.